src/sentry/eventstream/base.py

```diff
@@ -43,6 +42,13 @@ class GroupState(TypedDict):
 GroupStates = Sequence[GroupState]
 
 
+def get_post_process_queue(message_type: str) -> str:
+    if message_type == "transaction":
+        return "post_process_transactions"
```

Comment:
Has `post_process_transactions` queue been created? I think right now there's `post_process_performance`

Reply:
just fixed it in code. was an oversight that wasn't merged. do we need to circle back to rename it on an infrastructure level?

Reply:
I think this function adds an unnecessary level of indirection. Why mapping to a message type string first before the queue - would it make more sense to just pass the queue name? Especially since the message type is not really right and just seems to default to "error" anyway in most places.